integration-tests/src/tests/nearcore/sync_state_nodes.rs

```diff
@@ -395,3 +397,136 @@ fn sync_empty_state() {
         });
     });
 }
+
+/// Runs one node for some time, which dumps state to a temp directory.
+/// Start the second node which gets state parts from that temp directory.
+#[test]
+#[cfg_attr(not(feature = "expensive_tests"), ignore)]
+fn sync_state_dump() {
```

Comment:
> Like sometimes once a millisecond or so.

Switched to `actix_rt::time::sleep()` and now the delay between iterations is reliably `50ms`. Also demoted that log line to `debug!`

Reply:
yeah `debug!` sounds good. Although I don't think it's because of `std::thread::sleep(iteration_delay)`, since `nanosleep` should sleep for at least as long as you ask it to. Actually I think the frequency of it was because of the different threads all calling `check_new_epoch()` at the same time. No objections to using `actix_rt::time::sleep()` though. Don't think it matters much in this case which one you use but might as well.

And this test still fails for me every time with the same error I pasted. Does it pass for you? maybe it's platform dependent for some reason